cdap-docs/examples-manual/build.sh

```diff
@@ -154,9 +154,9 @@ function download_includes() {
   test_an_include 2176f426a50b2d68817cd7da7b7faffb ../../cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionApp.java
   test_an_include b3ad7dc7ddbb5b905145dbf15a5c9b93 ../../cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionMapReduce.java
   
-  test_an_include dc6519d102875a83621f262c33c61894 ../../cdap-examples/UserProfiles/src/main/java/co/cask/cdap/examples/profiles/UserProfiles.java
+  test_an_include 2260781c7cef2938aa36c946f3a34341 ../../cdap-examples/UserProfiles/src/main/java/co/cask/cdap/examples/profiles/UserProfiles.java
```

Comment:
So this tests whether the included file's MD5 matches that number? And the build fails if it is not the same? So every time someone changes a file, the docs build fails and you need to change the expected MD5 sums? Sounds like a lot of manual work, but I like it better than undetected broken literal includes. +1